### PR TITLE
Jr/188 andor config

### PR DESCRIPTION
**Pull request by jayich**
This branch works with our camera.

addresses: https://github.com/CampbellGroup/common/issues/188

### CODE REVIEW

**Comment by aransfor** [comment]
just saw this been busy, will check at the end of the week if it works

File: lib/servers/iXonServer/AndorVideo.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
-from config.andor_config import andor_config as config
+import andor_config as config
```
Comment (aransfor) [inline lib/servers/iXonServer/AndorVideo.py]: This is not the standard config format. If we are staying with the old format we should need no changes

File: lib/servers/iXonServer/AndorCamera.py [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import ctypes as c
-from config.andor_config import andor_config as config
 import os
+import andor_config as config
```
Comment (aransfor) [inline lib/servers/iXonServer/AndorCamera.py]: Same as lower comment

**Comment by aransfor** [comment]
I am confused by this commit, the camera code IS working with the current branch of common I thought? Anyways when I pull this branch I get an error that no module named andor_config

**Comment by jayich** [comment]
We will add a default andor_config.py here: https://github.com/CampbellGroup/common/tree/master/lib/config

and kill this file: https://github.com/CampbellGroup/common/blob/master/lib/servers/iXonServer/.gitignore
(source of confusion)

Sound good?

No config in lib/config and the .gitignore had us scratching our heads.

File: lib/servers/iXonServer/AndorVideo.py (hunk 1 above)
Comment (jayich) [inline lib/servers/iXonServer/AndorVideo.py]: the old format sounds great to me.  I though it changed here based on the .gitignore.

**Comment by aransfor** [comment]
i guess then why does the file still look for the config locally in lines 1 and 3? anyway the branch does not work on my end its still cant find the config

**Comment by jayich** [comment]
We were trying to guess at what setup you had based on the .gitignore and lack of default/example config in lib/config.  The camera server does not have Try/Except for the config to work out of the box.  We can add this later.

**Comment by jayich** [comment]
fixed here: https://github.com/CampbellGroup/common/pull/192

**Pull request closed without merging**